prism/src/main/java/org/apache/falcon/plugin/GraphiteNotificationPlugin.java

```diff
@@ -45,11 +45,11 @@ public void monitor(ResourceMessage message) {
         MetricNotificationService metricNotificationService =
                 Services.get().getService(MetricNotificationService.SERVICE_NAME);
         try {
-            String entityType = message.getDimensions().get("entity-type");
-            String entityName = message.getDimensions().get("entity-name");
+            String entityType = message.getDimensions().get("entityType");
```

Comment:
Actually, looks like the dimension names are not consistent.wf-instance-succeded looks like this:
2016-02-24 00:00:03,026 {Action:wf-instance-succeeded, Dimensions:{entity-type=PROCESS, nominal-time=2016-02-23T23:54Z, cluster=lhr1-emerald, start-time=2016-02-23T23:58Z, entity-name=FETL-ImpressionRC-Conversion, wf-id=0148158-160208053433534-oozie-oozi-W, run-id=0, wf-user=fetl, operation=GENERATE}, Status: SUCCEEDED, Time-taken:105676000000 ns}

The one sent by InstanceManagerProxy looks like this:
            @Dimension("entityType") @PathParam("type") final String type,
            @Dimension("entityName") @PathParam("entity") final String entity,
            @Dimension("colo") @QueryParam("colo") String colo,

Since you are using the former, I think you should stick with "entity-type" and "entity-name".